README.md

```diff
@@ -2,0 +1,17 @@
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
+* Commit and Pull Request Analysis with GitHub, BitBucket/Stash, GitLab (and also direct git repositories)
```

Comment:
Codacy has a fix for the issue: [Expected: dash; Actual: asterisk](https://app.codacy.com/gh/troubleshoot-codacy/codacy-test-fork-markdown-test/pullRequest?prid=7586344)
```suggestion
- Commit and Pull Request Analysis with GitHub, BitBucket/Stash, GitLab (and also direct git repositories)
```